packages/controls/src/widget_dragdrop.ts

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+    CoreDOMWidgetModel
+} from './widget_core';
+
+import {
+    DOMWidgetView, unpack_models, WidgetModel, WidgetView, JupyterLuminoPanelWidget, reject
+} from '@jupyter-widgets/base';
+
+import * as _ from 'underscore';
```

Comment:
Don't we also need jQuery for JupyterLab?

```javascript
import $ from 'jquery';
```

Reply:
the drag'n'drop code only uses browser api's so I think for this particular code it shouldn't be necessary.

Reply:
Isn't it needed here?

https://github.com/jupyter-widgets/ipywidgets/blob/b8890e8da3b57b1351545499533608dbe00893b1/packages/controls/src/widget_dragdrop.ts#L63

Reply:
Wow, good catch. I copied this code from the `widget_box.ts`. I am not even sure wether it's necessary. Will investigate, thanks!

Reply:
The line above should be enough to make it work in JupyterLab.